integrations/compute-systems/kubernetes.md

```diff
@@ -12,6 +12,9 @@ If you don't already have a Kubernetes cluster running, please see:&#x20;
 * Creating a cluster on AWS' [Elastic Kubernetes Service](https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html)
 * Creating a cluster on [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/docs/how-to/creating-a-zonal-cluster)
 
+If your operator requires large Python packages such as `torch` and `transformers`, we recommend
+allocating at least 32 GB of disk space to your Kubernetes node.
```

Comment:
Let's just be extra clear — to the servers that Kubernetes is running on. (Just so people don't mix up pods and nodes.)